apps/common-app/App.tsx

```diff
@@ -307,11 +307,13 @@ function navigate(
 
 function MainScreen({ navigation }: StackScreenProps<ParamListBase>) {
   useEffect(() => {
-    AsyncStorage.multiGet([OPEN_LAST_EXAMPLE_KEY, LAST_EXAMPLE_KEY]).then(
+    void AsyncStorage.multiGet([OPEN_LAST_EXAMPLE_KEY, LAST_EXAMPLE_KEY]).then(
       ([openLastExample, lastExample]) => {
         if (openLastExample[1] === 'true' && lastExample[1]) {
           navigate(navigation, lastExample[1]);
         }
+
+        return;
```

Comment:
Those are funny changes.

Reply:
Yep, this is because `eslint` for some reason started yelling at me and I wanted to avoid adding another ignores/warns 😅